osbuild/util/ostree.py

```diff
@@ -192,18 +192,16 @@ def pull_local(source_repo: PathLike, target_repo: PathLike, remote: str, ref: s
         *extra_args,
         repo=target_repo)
 
-
 def cli(*args, _input=None, **kwargs):
     """Thin wrapper for running the ostree CLI"""
     args = list(args) + [f'--{k}={v}' for k, v in kwargs.items()]
     print("ostree " + " ".join(args), file=sys.stderr)
-    subprocess.run(["ostree"] + args,
+    return subprocess.run(["ostree"] + args,
                    encoding="utf8",
-                   stdout=sys.stderr,
+                   stdout=subprocess.PIPE,
```

Comment:
ahh. I hadn't realized we'd need to make this change.. I honestly think this is the way it should be, but the osbuild devs might want stdout to still go to stderr to keep the same behavior. We can still achieve that goal if they ask us by just doing:

```
cp = subprocess.run(["ostree"] + args,
                   encoding="utf8",
                   stdout=sys.stderr,
                   stdout=subprocess.PIPE,\
                   input=_input,
                   check=True)
print(cp.stdout)
return cp
```

but let's wait and see what they say.